edx_reverification_block/xblock/tests/test_reverification_block.py

```diff
@@ -1,14 +1,15 @@
 """
 Tests the edX Reverification XBlock functionality.
 """
+
+import ddt
```

Comment:
Nitpick: This is a third-party import, as is `mock`. Could you move those imports to the second block of imports?